packages/mux-player/src/template.ts

```diff
@@ -114,8 +109,31 @@ export const ChromeRenderer = (props: MuxTemplateProps) => {
 };
 
 // prettier-ignore
-const MediaPlayButton = () => html`
-  <media-play-button>
+const SeekToLiveButton = (props: Partial<MuxTemplateProps>) => html`
+  <button 
+    aria-disabled="${props.inLiveWindow}"
+    in-live-window="${props.inLiveWindow}"
```

Comment:
could `aria-disabled` not be used in the CSS again? 
adding a `in-live-window` attr to a native element is invalid HTML no, maybe use the class attribute?